lib/clamp-link.in

```diff
@@ -157,7 +177,7 @@ do
     fi
 
     OBJS_TO_PROCESS+=( "$ARG" )
-  elif [[ "$ARG" =~ ^-l[^[:space:]]+$ ]] || [[ "$ARG" =~ [^[:space:]]+.a$ ]]; then
+  elif [[ "$ARG" =~ ^-l[^[:space:]]+$ ]] || [[ "$ARG" =~ [^[:space:]]+.a$ ]] || [[ "$ARG" =~ [^[:space:]]+.lo$ ]]; then
```

Comment:
it seems like .lo isn't an archive and it's more like an .o file, should this be added to line 173 instead?  Further down, it will try to open it up with ar if it's archive